examples/gym_ex/proxy/env.py

```diff
@@ -200,7 +180,7 @@ def _decode_percept(self, envelope: Envelope, expected_step_id: int) -> Message:
         """
         if envelope is not None:
             if envelope.protocol_id == PublicId.from_str("fetchai/gym:0.1.0"):
-                gym_msg = GymSerializer().decode(envelope.message)
+                gym_msg = GymMessage.serializer.decode(envelope.message)
```

Comment:
Note, the new usage is much more concise! No need to import the serializer individually; no need to instantiate anything.